### ISSUE TITLE
Hoare Logic

### ISSUE THREAD

**Issue by jonaprieto** [ProofSystem/Encyclopedia#90]
It's missing the proof systems of the Hoare Logic and its extensions.

**Comment by ceilican** [comment]
Yes. Would you like to submit them or suggest people who could submit them?
